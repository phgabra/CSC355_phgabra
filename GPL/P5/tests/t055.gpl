double a = 1.2345;

// these are all ok
int b = !a;
int c = !0.0;
int d = !.12346;
