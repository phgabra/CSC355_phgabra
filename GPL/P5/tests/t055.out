gpl.cpp::main()
  input file(tests/t055.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double a = 1.2345
int b = 0
int c = 1
int d = 0
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
